ui/package.json

```diff
@@ -35,8 +33,15 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^22.4.3",
     "jest-cli": "^22.4.3",
+    "jest-puppeteer": "^2.3.0",
+    "puppeteer": "^1.3.0",
     "react-test-renderer": "^16.3.1"
   },
+  "//": ["After https://github.com/facebook/jest/issues/6026 is fixed",
```

Comment:
Since we have a testing section, could we put this comment in the README instead?

Reply:
Changed some things, PTAL. Note this is how npm creator says we should write comments: https://stackoverflow.com/a/14221781

Reply:
This is good, I more just wanted to move the instructions for running to the README which you did.